### DISCUSSION TITLE
Make the profile page HR compatible with UI improvements

### DISCUSSION THREAD

**Discussion by Hembi** [codewars/codewars.com#3024] [Ideas]
Hi,

I would like to start a conversation about the UI of the profile page.
After days/months of practice everyone would like to share their profile with others to show achivements. The UI of the user profiles is hard to understand on first view. The language skills and rank diagram texts are very small, this structure is not informative. This page is OK for yourself to check your progress, but not enough good to share your success with others.

**Comment by fetm-opensource** [comment]
uuuh
